src/runtime/virtcontainers/kata_agent.go

```diff
@@ -364,6 +364,11 @@ func (k *kataAgent) init(ctx context.Context, sandbox *Sandbox, config KataAgent
 	k.kmodules = config.KernelModules
 	k.dialTimout = config.DialTimeout
 
+	configuredRequestTimeout := time.Duration(sandbox.config.ImageRequestTimeout) * time.Second
+	if configuredRequestTimeout > defaultRequestTimeout {
```

Comment:
We want to override regardless if this is bigger or smaller than the default one.

Reply:
Thanks, @fidencio . I will update.